templates/partials/summary/summary.html

```diff
@@ -6,30 +6,37 @@
   {% set summaryGroups = [] %}
 
   {%- for group in content.summary.groups if group.blocks -%}
-    {% do summaryGroups.append (
-      {
+
+      {% do summaryGroups.append (
+        {
         "groups": [
           {
             "groupTitle": group.title if group.title else None,
             "id": group.id if group.id else None,
             "rows": map_summary_item_config(
-                      group,
-                      content.summary.summary_type,
-                      content.summary.answers_are_editable,
-                      _("No answer provided"),
-                      _("Change"),
-                      _("Change your answer for:"),
-                      content.summary.calculated_question
-                    ),
-            "classes": "ons-u-mt-m" if loop.index > 1 else ""
+            group,
+            content.summary.summary_type,
+            content.summary.answers_are_editable,
+            _("No answer provided"),
+            _("Change"),
+            _("Change your answer for:"),
+            content.summary.calculated_question,
+            None,
+            ),
+            "classes": "ons-u-mt-m" if loop.index > 1 else "",
+            "placeholderText": group.links.empty_list_text,
+            "summaryLink": group.links.add_link,
           }
```

Comment:
The formatting needs updating. Same comment about keyword args applies for `map_summary_item_config`.

Reply:
Some nesting added, looks correct now, keywords added as well.